tests/end2end/playwright/globals.js

```diff
@@ -281,4 +281,19 @@ export async function expectParametersToContain(title, parameters, expectedParam
 
     return searchParams;
 }
+
+/**
+ * Create a GET request on a given url with Basic authentification admin:admin
```

Comment:
```suggestion
 * Create a GET request on a given URL with Basic authentication admin:admin
```

Common typo ;-) https://english.stackexchange.com/questions/7844/is-authentification-a-real-word